### PR TITLE
Setup babel transpiler

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

**Comment by wltsmrz** [comment]
LGTM

File: scripts/verify_ledger_json.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
+/* eslint-disable no-var */
 'use strict';
 
 var fs = require('fs');
-var Amount = require('../src/js/ripple').Amount;
```
Comment (sublimator) [inline scripts/verify_ledger_json.js]: Oops we missed this before huh

Comment (sublimator) [inline scripts/verify_ledger_json.js]: And the other scripts in bin too

**Comment by sublimator** [comment]
We are enforcing `no-var` now ?

```
 1 failing
  1) Server Handle close:
     AssertionError: 'function noOp(){}' === 'function noOp() {}'
      at Context.<anonymous> (/home/travis/build/ripple/ripple-lib/test/server-test.js:625:12)
      at callFn (/home/travis/build/ripple/ripple-lib/node_modules/mocha/lib/runnable.js:251:21)
      at Test.Runnable.run (/home/travis/build/ripple/ripple-lib/node_modules/mocha/lib/runnable.js:244:7)
      at Runner.runTest (/home/travis/build/ripple/ripple-lib/node_modules/mocha/lib/runner.js:374:10)
      at /home/travis/build/ripple/ripple-lib/node_modules/mocha/lib/runner.js:452:12
      at next (/home/travis/build/ripple/ripple-lib/node_modules/mocha/lib/runner.js:299:14)
      at /home/travis/build/ripple/ripple-lib/node_modules/mocha/lib/runner.js:309:7
      at next (/home/travis/build/ripple/ripple-lib/node_modules/mocha/lib/runner.js:248:23)
      at Immediate._onImmediate (/home/travis/build/ripple/ripple-lib/node_modules/mocha/lib/runner.js:276:5)
      at processImmediate [as _immediateCallback] (timers.js:358:17)
```

**Comment by clark800** [comment]
Yeah, the es6 linter config enforces no-var because there is no need for var anymore; let and const are superior in that the provide lexical scope, and const provides additional safety against mutation.

**Comment by wltsmrz** [comment]
LGTM

**Pull request merged**